src/mqt/yaqs/core/data_structures/networks.py

```
@@ -986,6 +987,15 @@ class MPO:
         Rotates the MPO tensors by swapping physical dimensions.
     """
 
+    _PAULI_2 = {
+        "I": np.eye(2, dtype=complex),
+        "X": np.array([[0, 1], [1, 0]], dtype=complex),
+        "Y": np.array([[0, -1j], [1j, 0]], dtype=complex),
+        "Z": np.array([[1, 0], [0, -1]], dtype=complex),
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)